cf-promises/cf-promises.c

```diff
@@ -500,9 +542,14 @@ static void ShowVariablesFormatted(EvalContext *ctx)
         free(varname);
     }
 
+    pcre_free(rx);
+
     SeqSort(seq, (SeqItemComparator)strcmp, NULL);
```

Comment:
Wow this cast is really dangerous, I believe the behaviour here is undefined, `strcmp()` is not compatible with `SeqItemComparator()`! Anyway It's old code so I ignore, and I grep'ed and it's all over the place. :-(